### PR TITLE
created a landing page of a Food News Magazine

### PR DESCRIPTION
**Pull request by nama0027**

### CODE REVIEW

**Comment by nama0027** [comment]
requested by Naushin Malik

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,184 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Finger+Paint&display=swap" rel="stylesheet">
+
+  <!-- Add icon library -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+  <div class="wrapper">
```
Comment (JessicaNordahl) [inline code/index.html:30]: I'm curious, how come you start with a class here?

Comment (JessicaNordahl) [reply code/index.html:30]: Oh, I see now -it's a wrapper -I haven't heard of one of those before. So it contains all of the body within this div

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,184 @@
...
+    </header>
+    <!-- Content -->
+    <section class="content">
+      <!-- Top Story -->
+      <section class="row">
+        <article class="top-article">
+          <a class= "rec-link" href="#">
+          <figure class="top-fig">
+            <img class="rec-img" src="./images/top-story.jpeg" alt="This week dish:Byriani" />
+            <p> <span> <b> 17 best biryani recipes</b></span> that are ambrosial enough to make you crave some more with each bite.</p>
```
Comment (JessicaNordahl) [inline code/index.html:60]: Nice and very user friendly to look at!

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,229 @@
...
+  opacity: 1;
+}
+.rec-link{
+  text-decoration: none;
+  color: #333;
+}
+.discription{
+ font-weight:bold;
+ text-align: center;
+}
```
Comment (JessicaNordahl) [inline code/style.css:116]: Smart way to use several classes and target them all to style

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,229 @@
...
+      
+      font-size: 300%;
+      margin-left: 25%;
+      width: 150%;
+      position:absolute; Top:150px; left:0;
+    }
+    to {
+      margin-left: 0%;
+      width: 100%;
+    }
```
Comment (JessicaNordahl) [inline code/style.css:216]: Ah, I was wondering how you managed to do the slide-ins. I see how it works here. Nice!

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,229 @@
...
+.fa-facebook, .fa-twitter, .fa-instagram {
+  color: white;
+  padding: 20px;
+  font-size: 30px;
+  width: 50px;
+  text-align: center;
+  text-decoration: none;
 }
+
+  /* media queries */
```
Comment (JessicaNordahl) [inline code/style.css:154]: I like that you use alot of "out-commented" information, like this

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,229 @@
...
+  padding: 10px;
+}
+.top-fig:hover p{
+  position:absolute; top:75%; left:0;
+  display: block;
+}
+.top-fig figcaption { 
+  position:absolute; Top:0; left:0; 
+  padding: 10px; 
+ }
```
Comment (JessicaNordahl) [inline code/style.css:74]: It is easy to read when you write all of the styling for one specific "thing" in your website. It gives me a sence of overview

**Review by JessicaNordahl — COMMENTED** [review]
Great work with the project. I like your soft colors and effects that both hovers and slides in. I'm impressed.

**Pull request closed without merging**